app/iot/devices.py

```diff
@@ -8,57 +9,57 @@
 
 # of course this code looks dumb, but imagine some real implementations of each method here
```

Comment:
It's not a good practice to leave comments like this in your code. If you want to indicate that the code is a placeholder or needs to be replaced with a 'real' implementation in the future, it's better to use a TODO comment. This way, it will be easier for you and other developers to find this place in the code later.